pkg/metamorph/client_test.go

```diff
@@ -432,12 +431,12 @@ func TestClient_SubmitTransactions(t *testing.T) {
 
 			require.Equal(t, tc.expectedStatuses, statuses)
 
-			if tc.expectedErrorStr == "" {
+			if tc.expectedErrorStr != "" {
 				require.NoError(t, err)
```

Comment:
Wondering why the test passes, but I think `require.NoError(t, err)` needs to be removed here

Reply:
Removed